share/man/nitunit.md

```diff
@@ -13,13 +13,16 @@ nitunit [*options*] FILE...
 Unit testing in Nit can be achieved in two ways:
 
 * using `DocUnits` in code comments
```

Comment:
Maybe leave "two ways" and change this as

    * using `DocUnits` in code comments or markdown files.